### ISSUE TITLE
Panic disabling timers in 0.9.0

### ISSUE THREAD

**Issue by crisidev** [jD91mZM2/xidlehook#46]
Hello, first of all thanks for this piece of software, it is amazing!

I think I found a bug in 0.9.0: I have 4 timers and if I try to disable them using the client, I get a panic when I disable the 4th. The order does not matter, eventually when I disable the last enabled timer, I always get the panic

```sh
xidlehook-client --socket /home/bigo/.xidlehook.sock control --action Disable --timer 1
xidlehook-client --socket /home/bigo/.xidlehook.sock control --action Disable --timer 3
xidlehook-client --socket /home/bigo/.xidlehook.sock control --action Disable --timer 2
xidlehook-client --socket /home/bigo/.xidlehook.sock control --action Disable --timer 0
```

```
thread 'main' panicked at 'timer error: the timer is shutdown, must be called from the context of Tokio runtime', /home/bigo/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-0.2.21/src/time/delay.rs:96:23
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
Full trace can be found here: https://gist.github.com/crisidev/f5a4c58bb4ce23fe8eb45b6c8fc5a745

**Comment by crisidev** [comment]
I just reinstalled version 0.8.2 and I can confirm it is not affected by the problem.

**Comment by jD91mZM2** [comment]
Oops, seems my move to Tokio wasn't tested enough. Thanks for the report!

**Comment by jD91mZM2** [comment]
Should be fixed in 0.9.1 :)

**Comment by crisidev** [comment]
I'll test it soon! Thanks a lot for the quick turnaround.
